spec/features/geo_viewer_spec.rb

```diff
@@ -4,14 +4,27 @@
   include PURLFixtures
   before do
     stub_purl_response_with_fixture(geo_purl)
+    visit_sandbox
+    fill_in_default_sandbox_form
+    click_button 'Embed'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.